scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -361,6 +427,63 @@ def get_args() -> argparse.Namespace:
             "The program will fail with a helpful message if any of the notebooks cannot be executed."
         ),
     )
+    parser.add_argument(
+        "--backend",
+        action="store",
+        default="fake_athens",
+        help=(
+            "Specify a backend to run the script against"
+        )
+    )
+    parser.add_argument(
+        "--provider",
+        action="store",
+        default="qiskit_fake_provider",
+        choices=["qiskit_ibm_runtime", "qiskit_fake_provider", "runtime_fake_provider"],
+        help=(
+            "Specify a provider to run notebook against [qiskit_ibm_provider, qiskit_fake_provider, runtime_fake_provider]"
+        )
+    )
+    parser.add_argument(
+        "--channel",
+        action="store",
+        default="ibm_quantum",
+        help=(
+            "Specify a channel for running the notebook against"
+        )
+    )
+    parser.add_argument(
+        "--token",
+        action="store",
+        default=None,
+        help=(
+            "IBM Cloud API key or IBM Quantum API token"
+        )
+    )
+    parser.add_argument(
+        "--url",
+        action="store",
+        default=None,
+        help=(
+            "The API URL to submit the notebook against"
+        )
+    )
+    parser.add_argument(
+        "--name",
+        action="store",
+        default=None,
+        help=(
+            "Name of the qiskit account to load"
```

Comment:
Can you please add an example?

Reply:
For the examples, would it be more helpful to link the QiskitRuntimeService page and explicitly say to reference that page for those args, to use the descriptions from that page, or to come up with unique descriptions?

Reply:
Linking to https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService makes sense. We want to make clear these are args for QiskitRuntimeService